### PR TITLE
Gag location message removal

### PR DESCRIPTION
**Pull request by senciel-dev**
Replaces the gag location message with a constant message to reduce user's confusion

### CODE REVIEW

**Comment by duogenesis** [comment]
Yes, please

**Comment by senciel-dev** [comment]
Ok, done

File: components/prospect-profile-screen.tsx [hunk 1]
```diff
@@ -26,8 +25,6 @@ import { ButtonWithCenteredText } from './button/centered-text';
 import { api } from '../api/api';
 import { cmToFeetInchesStr } from '../units/units';
 import { signedInUser } from '../App';
-import { IMAGES_URL } from '../env/env';
-import { randomGagLocation } from '../data/gag-locations';
```
Comment (duogenesis) [inline components/prospect-profile-screen.tsx:30]: Thanks for removing the unused `IMAGES_URL` import. Could you also remove the code in `../data/gag-locations`, which is now unused?

**Review by duogenesis — APPROVED** [review]
Looks good! Thanks for contributing

**Pull request merged**